lib/ansible/modules/cloud/amazon/aws_organization_units.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Kamil Potrec <kamilpotrec@googlemail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+module: aws_organization_units
+short_description: Manages AWS Organizations Units
+description:
+  - Creates and deletes AWS OrganizationalUnits.
+version_added: "2.10"
+author:
+  - Kamil Potrec (@p0tr3c)
+options:
+  name:
+    description:
+      - Name of the organizational unit
+        This can be full OU path such as "Prod/IT/Service_Desk".
```

Comment:
```suggestion
        This can be full OU path such as C(Prod/IT/Service_Desk).
```

See https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation
for more information